testsupport/wait/member.go

```diff
@@ -1259,14 +1259,14 @@ func (a *MemberAwaitility) WaitForConfigMap(t *testing.T, namespace, name string
 	return cm, err
 }
 
-// WaitForSecret waits until a Secret with the given name exists in the given namespace
-func (a *MemberAwaitility) WaitForSecret(t *testing.T, namespace, name string) (*corev1.Secret, error) {
```

Comment:
IMO keeping the namespace as a parameter would make this function reusable also for other use cases. I'm thinking for example at the spacerequest test which in the future may need to check secrets in other namespaces. But not a big issue.

Reply:
if needed, then we can change it... In the case it should be reusable, then it shouldn't be tied to any awaitility - just a standalone functino

Reply:
Makes sense!  

So `MemberAwaitility` is mainly designed to interact with the member operator namespaces only. I was thinking at the multi-cluster scenario/environment, where member is a cluster not only a namespace , but we don't support that yet in e2e tests.